tests/api.c

```diff
@@ -21038,6 +21038,7 @@ static int test_wc_ecc_signVerify_hash (void)
     #endif
     word32      siglen = ECC_BUFSIZE;
     byte        sig[ECC_BUFSIZE];
+    byte        adjustedSig[ECC_BUFSIZE+1];
```

Comment:
`adjustedSig[ECC_BUFSIZE]`(extra byte) is uninitialized but this might be good as valgrind will complain if we attempt to read more from the signature than we should.